### PR TITLE
FIX:  Refresh from notification #1723

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/f8xqdj5vym2q7071tcdppe3kdw

[download apk](http://filestorage.bluewallet.io:1488/1599706364.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/m999yuxka9727hu44qqa8zmwk8

[download apk](http://filestorage.bluewallet.io:1488/1599711145.apk)

**Comment by ncoelho** [comment]
See the loading, loads the blank state and then the content, all the time.

Screen Recording 2020-09-10 at 10 38 52

File: screen/wallets/transactions.js [hunk 1]
```diff
@@ -230,6 +230,7 @@ const WalletTransactions = () => {
     txs = txs.sort(function (a, b) {
       return b.sort_ts - a.sort_ts;
     });
+    txs.join(txs.reverse());
```
Comment (Overtorment) [inline screen/wallets/transactions.js:233]: i dont get it. `join` does nothing here, but the list does gets reversed

**Review by Overtorment — CHANGES_REQUESTED** [review]

File: screen/wallets/transactions.js (hunk 1 above)
Comment (marcosrdz) [reply screen/wallets/transactions.js:233]: dang..that was left over test code. good catch

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/ryyt02guxucttuw4pjb9nmvw8c

[download apk](http://filestorage.bluewallet.io:1488/1599748730.apk)

**Review by ncoelho — APPROVED** [review]

**Pull request merged**